test/test-component.js

```diff
@@ -256,3 +262,36 @@ describe('Nested Component instance', function() {
     });
   });
 });
+
+
+describe('Rendering exception', function() {
+  var el;
+
+  beforeEach(function(done) {
+    document.body.innerHTML = '';
+    el = document.createElement('breakable-app');
+    el.logError = sinon.spy();
+    document.body.appendChild(el);
+    window.requestAnimationFrame(function() {
+      done();
+    });
+  });
+
+  it('does not prevent component from initializing', function() {
+    expect(el.initialized).to.be.ok;
```

Comment:
Unfortunately we're not using `expect.js` here, but the lesser [Chai](http://chaijs.com/api/bdd/) (because wct automatically bundles it globally)